src/bootstrap/src/core/config/flags.rs

```diff
@@ -470,7 +470,9 @@ Arguments:
         versioned_dirs: bool,
     },
     /// Perform profiling and benchmarking of the compiler using the
-    /// `rustc-perf` benchmark suite.
+    /// `rustc-perf-wrapper` tool.
+    ///
+    /// You need to pass arguments after `--`, e.g.`x perf -- cachegrind`.
```

Comment:
The example command mismatch with the one in this README: https://github.com/rust-lang/rust/blob/151986f493bce146db808db2bbb2b29fa6e0049c/src/tools/rustc-perf-wrapper/README.md?plain=1#L3

Reply:
Good catch, thanks!